### PR TITLE
Improve the API of  LayerNormalization

### PR DESCRIPTION
**Pull request by Beacontownfc**
The ` LayerNormalization` didn't work as expected, it throws an `System.NullReferenceException` during runtime.

### CODE REVIEW

File: src/TensorFlowNET.Core/APIs/tf.nn.cs [hunk 1]
```diff
@@ -14,8 +14,10 @@
    limitations under the License.
 ******************************************************************************/
 
+using System.Xml.Linq;
 using Tensorflow.Operations;
 using Tensorflow.Operations.Activation;
+//using static System.Formats.Asn1.AsnWriter;
```
Comment (AsakusaRinne) [inline src/TensorFlowNET.Core/APIs/tf.nn.cs:20]: Is this using necessary?

File: src/TensorFlowNET.Core/APIs/tf.nn.cs [hunk 2]
```diff
@@ -125,6 +127,22 @@ public Tensor relu(Tensor features, string name = null)
                     is_training: is_training,
                     name: name,
                     exponential_avg_factor: exponential_avg_factor);
+            public Tensor batch_normalization(Tensor x,
```
Comment (AsakusaRinne) [inline src/TensorFlowNET.Core/APIs/tf.nn.cs:130]: Please add some simple comments of this API since it's exposed to users

**Review by AsakusaRinne — CHANGES_REQUESTED** [review]

File: src/TensorFlowNET.Core/APIs/tf.nn.cs (hunk 1 above)
Comment (Beacontownfc) [reply src/TensorFlowNET.Core/APIs/tf.nn.cs]: Not necessary, deleted.

File: src/TensorFlowNET.Core/APIs/tf.nn.cs (hunk 2 above)
Comment (Beacontownfc) [reply src/TensorFlowNET.Core/APIs/tf.nn.cs:141]: Done

**Review by AsakusaRinne — APPROVED** [review]

**Pull request merged**